package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "steampipe-cloud-zapier",
+  "version": "0.0.10",
+  "description": "Steampipe Cloud app for the Zapier platform.",
```

Comment:
What should these fields contain? Any recommendations from Zapier, or other popular apps?

Reply:
Thanks, @cbruno10; I will update the `author`. Below is the sample of `package.json` provided in Zapier documentation.
https://platform.zapier.com/cli_tutorials/cli-branding